src/test/java/htmlFindElements.java

```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class htmlFindElements {
```

Comment:
Name for java classes should start from Capital letter.

Reply:
done